src/NServiceBus.Core/DataBus/DataBusExtensions.cs

```diff
@@ -30,5 +33,41 @@ public DataBusExtensions(SettingsHolder settings)
             : base(settings)
         {
         }
+
+        /// <summary>
+        /// Configures additional deserializers to be considered when processing data bus properties. Can be called multiple times.
+        /// </summary>
+        public DataBusExtensions AddDeserializer<TSerializer>() where TSerializer : IDataBusSerializer, new()
+        {
+            var serializer = (TSerializer)Activator.CreateInstance(typeof(TSerializer));
+
+            return AddDeserializer(serializer);
+        }
+
+        /// <summary>
+        /// Configures additional deserializers to be considered when processing data bus properties. Can be called multiple times.
+        /// </summary>
+        public DataBusExtensions AddDeserializer<TSerializer>(TSerializer serializer) where TSerializer : IDataBusSerializer
+        {
+            Guard.AgainstNull(nameof(serializer), serializer);
+
+            var deserializers = Settings.Get<List<IDataBusSerializer>>(Features.DataBus.AdditionalDataBusDeserializersKey);
+
+            if (deserializers.Any(d => d.ContentType == serializer.ContentType))
+            {
+                throw new ArgumentException($"Deserializer for content type  {serializer.ContentType} already registered.");
```

Comment:
Same as above, wrapping in quotes.

```suggestion
                throw new ArgumentException($"Deserializer for content type  '{serializer.ContentType}' is already registered.");
```